drivers/led/lp5562.c

```diff
@@ -1001,7 +1031,7 @@ static int lp5562_led_init(const struct device *dev)
 		return -ENODEV;
 	}
 
-	if (lp5562_enable(dev) != 0) {
+	if (lp5562_enable(dev, true) != 0) {
 		LOG_ERR("LP5562 Enable failed");
```

Comment:
I think you can get rid of this log message. Indeed all the error cases are already logged in `lp5562_enable()` . So this generic message brings nothing more.